src/scss/base.scss

```diff
@@ -1,17 +1,44 @@
-.orange {
-  background: $orange;
-  width: 100px;
-  height: 100px;
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 92vh;
+  border-top: 8px solid $orange;
+  margin: 0;
```

Comment:
you have to add this because of the browser's default styles, to avoid overriding this in every page you create you have to add a reset css file. there are a lot of them, we created one that makes all the elements look the same. you can find it here https://codepen.io/SKOLZ/pen/pWJjRL